packages/xo-server-web-hooks/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "xo-server-web-hooks",
+  "version": "0.0.0",
+  "license": "ISC",
```

Comment:
License not consistent with the README.

Use:
- `ISC`: for all code not-related to XO like generic libraries
- `AGPL-3.0`: for all XO code